src/main/java/com/refinedmods/refinedstorage/util/BlockUtils.java

```diff
@@ -33,7 +31,7 @@
         return ActionResultType.SUCCESS;
     }
 
-    public static <T extends BaseBlock> ActionResultType changeBlockColor(Map<DyeColor, RegistryObject<T>> map, BlockState state, ItemStack heldItem, World world, BlockPos pos, PlayerEntity player) {
+    public static <T extends BaseBlock> ActionResultType changeBlockColor(ColorMap<T> map, BlockState state, ItemStack heldItem, World world, BlockPos pos, PlayerEntity player) {
```

Comment:
Can we move this method to `ColorMap` too possibly?

Reply:
We can but the other part of this method cannot be. Seems weird to split them.

Reply:
Maybe move both then